compile.sh

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env bash
+usage() {
+    echo '                Please export USE_TESTENV_PROPERTIES, JDK_VERSION and JDK_IMPL before running the script locally.'
+
+}
+if [ $(uname) = AIX ] || [ $(uname) = SunOS ] || [ $(uname) = *BSD ]; then
+    MAKE=gmake
+else
+    MAKE=make
+fi
+if [ $USE_TESTENV_PROPERTIES == true ]; then
+    while read line; do
+        export $line
+    done <./testenv/testenv.properties
+    if [ $JDK_IMPL == "openj9" ]; then
+        repo=JDK${JDK_VERSION}_OPENJ9_REPO
```

Comment:
Probably rename repo, branch to repoTemp and branchTemp. And then rename repo2 and branch2 to repo and branch. It's just my personal preference, which feels like more clear by naming.